doc/dash/dash-sonic-hld.md

```diff
@@ -674,6 +675,12 @@ key                      = DASH_PA_VALIDATION_TABLE:vni; ENI and VNI as key;
 addresses                = list of addresses used for validating underlay source ip of incoming packets. 
 ```
 
+DASH PA_VALIDATION_TABLE is used only for PL outbound direction. PA address can be either IPV4 or IPV6.
+
+Total PAs per MSEE would be 64 and if there are 64 MSEEs per region, there would be 4K PA_VALIDATION entries.
```

Comment:
will be better to mention the scale is based on 400G card.

Reply:
Done